.github/workflows/ci.yml

```diff
@@ -201,10 +203,12 @@ jobs:
         run: |
           docker run --rm --pull never solidproject/community-server:edge --version
       - 
-        name: "Test docker image 'latest'"
+        name: "Test all docker built image tags"
         if: startsWith(github.ref, 'refs/tags/v')
         run: |
-          docker run --rm --pull never solidproject/community-server:latest --version
+          jq -c '.[]' <<< "${{ steps.meta.outputs.tags }}" | while read i; do
+            docker run --rm --pull never solidproject/community-server:$i --version
+          done;
```

Comment:
I'm going to need a comment to understand this.

Reply:
So I'm trying to test each generated tag. But I'm having a hard time looping over the `steps.meta.out.tags` object. I am still working on it, but might end up veering of into separate test jobs, defined by a matrix.

Reply:
Explained in comment now